### ISSUE TITLE
Update code in doc/ directory to use non-deprecated PB stuff

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#10371]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)| @puzzlement reported|
|-|-|
|Trac ID|trac#545|
|Type|defect|
|Created|2004-03-06 07:53:00Z|

<details><summary>Searchable metadata</summary>

```
trac-id__545 545
type__defect defect
reporter__hypatia hypatia
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__pb pb
keywords__documentation documentation
time__1078559580000000 1078559580000000
changetime__1476081024986375 1476081024986375
version__ 
owner__ 
cc__slyphon cc__hypatia cc__cinober
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)|@puzzlement commented|
|-|-|

```
#!html
<pre>
Also yuck, as it probably means reading and understanding the PB docs. I haven't
got a grep for this one, but there's at least 10 files that need to be changed
and checked for this one.

This one will supersede 437, which was to update the examples.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)|@puzzlement commented|
|-|-|

```
#!html
<pre>
Resolved by removing a bunch of examples that weren't used anywhere, and by
trivially updating the rest.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="markrwilliams's avatar" src="https://avatars.githubusercontent.com/u/704247?s=50" width="50" height="50">](https://github.com/markrwilliams)<a name="note_4"></a>|@markrwilliams commented|
|-|-|

In changeset 5c1777777f85aaf799c2269f9f5dddcf0d59803e
```
#!CommitTicketReference repository="" revision="5c1777777f85aaf799c2269f9f5dddcf0d59803e"
Merge pull request #545 from twisted/8817-incremental

Author: hawkowl
Reviewer: glyph, markrwilliams
Fixes: #8817

Replace t.p.versions with incremental
```
